pkg/eval/platform/platform.go

```diff
@@ -23,7 +23,21 @@ import (
 // [`GOOS`](https://pkg.go.dev/runtime?tab=doc#pkg-constants) constant.
 // This is read-only.
 
+//elvdoc:var is-unix
+//
+// Whether or not the platform is UNIX-like (i.e., not a specific flavor such
```

Comment:
The part in the parentheses is a bit confusing. Instead, enumerate all the OSes that are now considered Unix.

Since Elvish only supports Linux, macOS and {Free,Net,Open}BSD, listing those would be sufficient.

Reply:
Okay, but I don't see any reason why other UNIX-like platforms such as AIX, DragonFly and Illumos (the Solaris fork) shouldn't also work. However, I can appreciate why you might not want to give that impression since you probably haven't tested elvish on any of those platforms.